docs/proposals/docs-process.md

```diff
@@ -0,0 +1,94 @@
+
+## Overview of docs migration and how release process will change
+
+Right now, the subdirectory of kubernetes/docs acts as the HEAD for the docs, which is used to populate new releases, as well as the basis for cherrypicking changes into docs for old releases. However, consider a "roaming HEAD" situation, such that the new release is always treated as the basis for populating new releases and  cherrypicks. In otherwords, whatever information is `http://kubernetes.io/vNEWEST/` is HEAD, and we no longer have a dependency on an underlying “master” docset. 
+
+New releases can be developed under `http://kubernetets.io/vNEXT`. We can do this because there is an extensive set of steps to get any one particular folder full of docs to be the default. Or, at worst, we can make a vNEXT branch of the `kubernetes/kubernetes.github.io` repo and do vNEXT work there, culminating in a pull request at release time. But this would only be necessary if we don’t want users to ever see what vNEXT docs look like. I personally vote for doing our work out in public. We can always mark those docs "beta" -- and keeping them in the master branch means that we get to use all the great github.io staging tools. 
+
+## Replacing of munging scripts
+
+- Inline/top-of-page Table of Contents can be done with JavaScript, e.g.: http://jsfiddle.net/s8m2t/8/
+- Header/footer insertion can be done by simply using the native layouts functionality in Jekyll
+- Linking to specific versions of docs can be done with inline variables in the URL, e.g. /docs/{{ version }}/hello-world.md
+- Replacement of file extensions in link paths should not be necessary as public Jekyll URLs are "friendly" (extension-free)
+- Other pre/post-processing scripts for docs?
+
+## Adapting the current docs process for releases
+
+Nuts and bolts: How would a pure-GitHub Pages process work? Let’s rewrite this doc that we provide to the community, which covers [how to contribute to our docs for a K8s release](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/update-release-docs.md).
+
+### Adding a new docs collection for a release
+
+Whenever a new release series (release-X.Y) is cut from master, we push the corresponding set of docs to `http://kubernetes.io/vX.Y/docs`. The steps are as follows:
+
+* Create a `_vX.Y` folder in the `kubernetes.github.io` repo.
+
+* Add vX.Y as a valid collection in `_config.yml`.
+
+* Create a new `_includes/nav_vX.Y.html` file with the navigation menu. This can be a copy of `_includes/nav_vX.Y-1.html` with links to new docs added and links to deleted docs removed. Update _layouts/docwithnav.html to include this new navigation html file. Example PR: [#16143](https://github.com/kubernetes/kubernetes/pull/16143).
+
+* [Pull docs from release branch](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/update-release-docs.md#updating-docs-in-gh-pages-branch) in _vX.Y folder.
```

Comment:
I think this is a proposed change to that section of the existing process -- that's the title of a section of the old instruction.